### DISCUSSION TITLE
application-command-autocomplete

### DISCUSSION THREAD

**Discussion by giscus** [brainboxdotcc/dpp-comments#45] [General]
# application-command-autocomplete

A lightweight C++ Discord API library supporting the entire Discord API, including Slash Commands, Voice/Audio, Sharding, Clustering and more!

https://dpp.dev/application-command-autocomplete.html

**Comment by JAHammy** [comment]
This code doesn't seem to function anymore. When I run it I keep getting a loader where the autocomplete is supposed to be, and the console output just reads "Error 50035: Invalid Form Body" over and over again.
